learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md

```diff
@@ -105,7 +115,13 @@ Follow the steps below to write a simple program in a Ballerina package, which p
 	    target/bin/helloworld.jar
     ```
 
-5. Execute the `bal run` command to run the program. You view the output below.
+5. Execute the command below to run the program. You view the output below.
+
+    ```bash
+    bal run
+    ```
+
+    You view the output below.
```

Comment:
"You view the output below" segment is repeated here.

Reply:
Fixed.